frontend/Data.js

```diff
@@ -101,4 +101,21 @@ export default class Data {
       return e.response.data.errors
     }
   }
+  async createComment(comment) {
+    try {
+      const response = await this.api('post/' + comment.post, 'POST', comment)
+      if (response.status === 201) {
+        return response
+      } else if (response.status === 400) {
+        return response.json().then(data => {
+          return data.errors
+        })
+      } else {
+        throw new Error(response.message)
+      }
+    } catch (e) {
+      console.log(e)
+      return e.response.data.errors
+    }
+  }
```

Comment:
this is again an amazing technique because in this case, there is only one place to adjust request method, URL, headers and API stuff. I presume this file is responsible for handling anything related to API. Very cool ! I should definitely do it like that next time.

Reply:
Cheers man